portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/FragmentsExportImport.testcase

```diff
@@ -1629,4 +1629,135 @@ definition {
 		}
 	}
 
+	@description = "This is a test case for LPS-133280. View if the changes of fragment collection on content page still persist after imported."
+	@priority = "4"
+	test ViewChangesOfFragmentCollectionOnContentPageAfterImported {
+		task ("Go to System Settings on Control Panel") {
```

Comment:
Add property to run on localhost
`property test.name.skip.portal.instance = "FragmentsExportImport#ViewChangesOfFragmentOnContentPagePersistAfterOverwriteTheFragment";`